README.md

```diff
@@ -1,4 +1,15 @@
+answer-one
+# About Me
+My name is Brittany Anderson. I am currently a Contract Administrator for a construction company. I have experience in    contract management, procurement, vendor compliance, finance and customer service. I have worked in both the public and private sectors managing a high volume of complex contracts as well as procuring a wide variety of goods and services. 
+## For Fun: 
+            * Yoga
+            * Baking
+            * Watching trash television (pretty much everything on BravoTV)
+            * Traveling
+            * Trying new wines/beers with my husband
+=======
 ## Profiles
   * [GitHub] (https://github.com/BMAnderson1913
   ## Technical Skills
   MMARS, COMMBUYS, Microsoft Office (Excel, Powerpoint, Word, Outlook), Viewpoint
+  master
```

Comment:
These labels are given to use by git during a conflict so we know where each change comes from. We'd want to remove them as part of resolving the conflict since they are part of the conflict, not our final file. In fact, leaving them in there could cause our code to break, crashing the production system.